src/System.Private.Xml/src/System/Xml/Xsl/Xslt/Scripts.cs

```diff
@@ -130,7 +130,7 @@ public ScriptClass GetScriptClass(string ns, string language, IErrorHelper error
                 compilerInfo = CodeDomProvider.GetCompilerInfo(language);
                 Debug.Assert(compilerInfo != null);
             }
-            catch (ConfigurationException)
+            catch (SystemException)
```

Comment:
No, this class is internal. The exception being caught here is `ConfigurationErrorsException` which used to inherit from `ConfigurationException` (which doesn't exist now). Currently, the exception being thrown in GetCompilerInfo method is still `ConfigurationErrorsException` defined [here](https://github.com/dotnet/corefx/blob/ab0c04f10c8094731d13de6f0ceaa71dd776aa00/src/System.CodeDom/src/System/CodeDom/Compiler/CodeDomProvider.cs#L216) which inherits directly from `SystemException`.